andela/AndelaEatsUI
# #161079580 Implement edit Engagement UI

#### What does this PR do?
Implement `Edit` Engagement UI

#### Description of Task to be completed?
Ensure there is a UI for Admin to Edit Engagements

#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/engagements)
- You should see a button with `Edit`
- Click the button and a modal will be displayed

#### What are the relevant pivotal tracker stories?
- [#161079580](https://www.pivotaltracker.com/n/projects/2178749/stories/161079580)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1278" alt="screen shot 2018-11-15 at 9 03 51 pm" src="https://user-images.githubusercontent.com/21138774/48579235-2f7edf00-e91c-11e8-99fb-2a8e5fa6caeb.png">
